.eslintrc.yaml

```diff
@@ -20,3 +20,9 @@ overrides:
     rules:
       max-nested-callbacks: off
       mocha/no-exclusive-tests: error
+  - files: "**/*.ts"
+    parserOptions:
+      ecmaVersion: 2020
+      sourceType: 'module'
```

Comment:
removes the syntax error for `import`, but still syntax error for `type`. Might be worth it to add `@typescript-eslint/parser` or something, but felt very out of scope